code/game/gamemodes/meteor/meteors.dm

```diff
@@ -295,8 +295,8 @@
 
 /obj/effect/meteor/supermatter
 	name = "supermatter shard"
-	icon = 'icons/obj/engine.dmi'
-	icon_state = "darkmatter"
+	icon = 'icons/obj/supermatter.dmi'
+	icon_state = "darkmatter_old"
```

Comment:
I don't like the choice of icon state name, if an icon is old/deprecated then we can just remove it. Would prefer it to be called something like "darkmatter_meteor". Same for the other sprite you added "_old" to.

Pretty minor nitpick though.